### PR TITLE
DBOOK-8: Global styles

### PR DESCRIPTION
**Pull request by mariaozamiz**
The purpose of this task was to add daily used global styles. You can add this as a page, which can be linked from the Theme creation section, or it can be embedded there.

### CODE REVIEW

File: pages/global-styles.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  ${normalize}
+
+  body {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",
+    "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
+    "Helvetica Neue", sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    scroll-behavior: smooth;
+  }
```
Comment (ghost) [inline pages/global-styles.mdx]: suggestion: we could include also this text settings

```
	text-rendering: optimizeLegibility;
	font-feature-settings: "liga","kern"
	font-variant-numeric: proportional-nums;
```

File: pages/global-styles.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  box-sizing: border-box;
+  }
+
+  @media (prefers-reduced-motion: reduce) {
+    * {
+      transition: none !important;
+      animation: none !important;
+      scroll-behavior: auto !important;
+    }
+  }
```
Comment (ghost) [inline pages/global-styles.mdx]: suggestion: I wouldn't add these styles to all elements, I prefer the approach of using them only where we need it.

**Review by ghost — APPROVED** [review]
Nice!

File: pages/global-styles.mdx (hunk 1 above)
Comment (mariaozamiz) [reply pages/global-styles.mdx]: done!

**Pull request merged**